<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDialog</class>
 <widget class="QDialog" name="ReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BReport - Visualizador de reportes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="BReportResources.qrc">
    <normaloff>:/iconos/iconoReport</normaloff>:/iconos/iconoReport</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="visualizador">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pdfButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Guardar como PDF</string>
       </property>
       <property name="icon">
        <iconset resource="BReportResources.qrc">
         <normaloff>:/iconos/iconoPDF</normaloff>:/iconos/iconoPDF</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="odtButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Guardar como ODT</string>
       </property>
       <property name="icon">
        <iconset resource="BReportResources.qrc">
         <normaloff>:/iconos/iconoODT</normaloff>:/iconos/iconoODT</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Imprimir</string>
       </property>
       <property name="icon">
        <iconset resource="BReportResources.qrc">
         <normaloff>:/iconos/iconoImpresora</normaloff>:/iconos/iconoImpresora</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="BReportResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ReportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
